File: javaCode/lecture06/Lecture06Main.java
package javaCode.lecture06;

import java.util.Arrays;
import java.util.List;

public class Lecture06Main {

    public static void main(String[] args) {
        List<Long> numbers = Arrays.asList(1L, 2L, 3L);
        for (Long number : numbers) {
            System.out.println(number);
        }

        for (int i = 0; i <= 100; i++) {
            System.out.println(i);
        }

        for (int i = 3; i >= 1; i--) {
            System.out.println(i);
        }

        for (int i = 1; i <= 5; i += 2) {
            System.out.println(i);
        }
    }
}

File: lecture06/Lecture06Main.kt
package lecture06

fun main() {

    val numbers = listOf(1L, 2L, 3L)

    for (number in numbers) {
        println(number)
    }

    for (i in 0..100) {
        println(i)
    }

    for (i in 3 downTo 0) {
        println(i)
    }

    for (i in 1..5 step 2) {
        println(i)
    }
}